changehc/delphi_changehc/update_sensor.py

```diff
@@ -113,13 +109,13 @@ def __init__(self,
 
```

Comment:
V minor: can we also change `CDCSensorUpdator` to `CDCSensorUpdater`? Updator makes me think of giant robots, for some reason 😄

Reply:
> Updator makes me think of giant robots, for some reason

You act like this is a problem :thinking: lol just changed this in a new commit

Reply:
Haha, you're not wrong! Thanks!